src/services/ImageService.ts

```diff
@@ -0,0 +1,72 @@
+import { injectable } from 'tsyringe';
+import { ImageMetadata } from '@map-colonies/mc-model-types';
+import { ConnectionManager } from '../DAL/ConnectionManager';
+import { ImageDataRepository } from '../DAL/ImageDataRepository';
+import { ImageData } from '../entity/ImageData';
+
+@injectable()
+export class ImagesService {
+  private repository?: ImageDataRepository;
+  public constructor(private readonly connectionManager: ConnectionManager) {}
+
+  public async get(id: string): Promise<ImageMetadata> {
+    const repository = await this.getRepository();
+    const image = await repository.get(id);
+    if (!image) {
+      //TODO: replace with custom error type
+      throw new Error('image data was not found');

```

Comment:
return data (id) on error log

Reply:
error handling here is temporary will be handled in separate task/pr